RIOT-OS/RIOT
# drivers/a*: model modules in Kconfig

### Contribution description
Continuing with the module dependency modelling in Kconfig, this is the first of a series of PRs that will start modelling the high-level drivers.  Given the amount of drivers I will be splitting this into multiple PRs. I've started in alphabetical order, this contains most of the one starting with `a`. Some network devices are not in this PR given that many test applications bring the GNRC stack, and that's not yet modelled.

### Testing procedure
- Check that the dependencies are correctly modelled, comparing to the current Makefiles.
- Check that the test applications of the correspondent drivers are added to the list of the CI
- Green CI

### Issues/PRs references
None
